coderd/schedule/mock.go

```diff
@@ -23,12 +23,12 @@ func (m MockTemplateScheduleStore) Get(ctx context.Context, db database.Store, t
 	return NewAGPLTemplateScheduleStore().Get(ctx, db, templateID)
 }
 
-func (m MockTemplateScheduleStore) Set(ctx context.Context, db database.Store, template database.Template, opts TemplateScheduleOptions) (database.Template, error) {
```

Comment:
Please revert this file; this was a bad refactor from my side which I fixed, but it seems you must've had a conflict and restored the changes.

Reply:
This was reverted already.